### DISCUSSION TITLE
Latex support

### DISCUSSION THREAD

**Discussion by ifryan** [alangrainger/share-note#98] [Vote for upcoming features]
the Latex formula show nothing on the shared page 
[check out this link]( https://share.note.sx/govcrsta#cKusFPXUQ2ZN197cvwfOhDi65yT5oYTJPOk24FGo88A)

**Comment by alangrainger** [comment]
Hi, did you follow the steps in the docs?

https://docs.note.sx/troubleshooting#mathjax--latex

Here's what your code looks like after following those steps:

https://share.note.sx/wfl5qxii#j+bzbjvejUgj/axz38+K00zrQJ1zkZaoJisVnICUCm4
